html/terms_of_use.html.j2

```diff
@@ -90,8 +90,8 @@
                 </div>
                 <div class="col footer-c3">
                     <p class="footer-c3-1">ACCESSIBILITY</p>
-                    <p><a target="_blank" href="https://docs.csc.fi/cloud/rahti/">Accessibility statement</a></p>
-                    <p><a target="_self" href="#">Terms of use </a></p>
```

Comment:
I think it is better to keep the links on the same page as "_self" if we are on the terms of use page and we click the link to Terms of use, the browser should remain on the same page instead of opening a new page,

Reply:
Indeed. I am keeping it to "_self" everywhere other than the index page. Otherwise too many opening of new tabs.